src/buckets/components/context/csvUploaderProvider.tsx

```diff
@@ -155,24 +194,29 @@ export const CsvUploaderProvider: FC<Props> = React.memo(({children}) => {
             const resp = postWrite({
               data: chunk,
               query: {org: org.name, bucket, precision: WritePrecision.Ns},
-            }).then(() => {
+            }).then(v => {
               const percent = (++progress / counter) * 100
               setProgress(Math.floor(percent))
+              return v
             })
             pendingWrites.push(resp)
             counter++
           }
-
           chunk = ''
-          Promise.all(pendingWrites).finally(() => {
-            setUploadState(RemoteDataState.Done)
-          })
+          Promise.all(pendingWrites)
+            .then(values => {
+              if (values.find(v => v.status >= 400)) {
+                throw new Error(
+                  `The CSV data could not be written to the bucket`
```

Comment:
```suggestion
                  `Looks like some of the CSV data could not be written to the bucket. Please make sure that CSV was in Annotated Format https://docs.influxdata.com/influxdb/cloud/reference/syntax/annotated-csv/`
```

Reply:
Obviously, that's a bit long-winded, but there might be a situation where some of the data successfully writes and some doesn't